specs/trip_dispatch_spec.rb

```diff
@@ -89,4 +91,84 @@
       passenger.trips.must_include trip
     end
   end
+
+  describe "request_trip" do
```

Comment:
Missing test:
You also need to test what happens when you try to request trips when there are no available drivers.